docs/UserGuide.md

```diff
@@ -154,32 +212,31 @@ Format: `s-find KEYWORD [MORE_KEYWORDS] ...`
 Examples:
 * `s-find BSBBT` returns `BSBBT`'s sales data.
 * `s-find BSBBT BSBM` returns `BSBBT`'s sales data and `BSBM`'s sales data.<br>
- 
-#### 1.4 Ranking the list of drinks sold : `s-rank`
-Shows a ranked list of all types of drinks sold in the Sales Tracker. The list of drinks
-is ranked from the most popular to least popular.
-  
-Format: `s-rank`
-
- 
+
+
```

Comment:
For line 215 ~216, A similar problem here, you may want to change back to what it was. Thanks

Reply:
Noted for all, will change back to the original version!